### PR TITLE
Update team members documentation

### PR DESCRIPTION
**Pull request by Uk1288**
### What

Update Team Membership API to include adding and removing users from teams using organization membership ID.

### Why

Update documentation to match existing API

### Screenshots

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [x] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [ ] Description links to related pull requests or issues, if any.

#### Content
- [ ] Redirects have been added for moved, renamed, or deleted pages. This requires a separate PR in the [`terraform-website` repository](https://github.com/hashicorp/terraform-website) `redirects.next.js` file.
- [x] API documentation and the API Changelog have been updated. 
- [ ] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [ ] Pages with related content are updated and link to this content when appropriate.
- [ ] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [ ] New pages have metadata (page name and description) at the top.
- [ ] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [x] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [ ] UI elements (button names, page names, etc.) are bolded.
- [x] The Vercel website preview successfully deployed.

#### Reviews
- [x] I or someone else reviewed the content for technical accuracy.
- [x] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

File: website/docs/cloud-docs/api-docs/team-members.mdx [hunk 1]
```diff
@@ -93,9 +93,58 @@ curl \
   https://app.terraform.io/api/v2/teams/257525/relationships/users
 ```
 
-## Delete a User from Team
+## Add a User to Team (With organization membership ID)
 
-This method removes multiple users from a team. Both users and teams must already exist. This DOES NOT delete the user; it only removes them from this team.
+This method adds multiple users to a team using the organization membership ID.
```
Comment (nfagerlund) [inline website/docs/cloud-docs/api-docs/team-members.mdx:98]:
```suggestion
This method adds multiple users to a team using the organization membership ID. Unlike the user ID method, the user doesn't need to exist yet; they only need to have been invited to the organization.
```

**Review by nfagerlund — APPROVED** [review]
Seems good, and matches my understanding of the API. My only (optional) suggestion is to mention _why_ one would want to choose this method — probably only needs a note in the "add" endpoint, the delete one is fine as is.

File: website/docs/cloud-docs/api-docs/team-members.mdx [hunk 2]
```diff
@@ -93,9 +93,58 @@ curl \
...
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request POST \
+  --data @payload.json \
+  https://app.terraform.io/api/v2/teams/257525/relationships/organization-memberships
+```
+
+## Delete a User from Team (With user ID)
+
+This method removes multiple users from a team using the user ID. Both users and teams must already exist. This DOES NOT delete the user; it only removes them from this team.
```
Comment (mgarrell777) [inline website/docs/cloud-docs/api-docs/team-members.mdx:147]:
```suggestion
This method removes multiple users from a team using the user ID. Both users and teams must already exist. This method only removes a user from this team. It does not delete that user overall.
```

Suggest rewording to adding "method" after "This" to give "This" a subject. Change unneeded capitalization.

File: website/docs/cloud-docs/api-docs/team-members.mdx [hunk 3]
```diff
@@ -141,3 +190,52 @@ $ curl \
   --data @payload.json \
   https://app.terraform.io/api/v2/teams/257525/relationships/users
 ```
+
+## Delete a User from Team (With organization membership ID)
+
+This method removes multiple users from a team using the organization membership ID. This DOES NOT delete the user; it only removes them from this team.
```
Comment (mgarrell777) [inline website/docs/cloud-docs/api-docs/team-members.mdx:196]:
```suggestion
This method removes multiple users from a team using the organization membership ID. This method only removes a user from this team. It does not delete that user overall.
```

Suggest rewording to adding "method" after "This" to give "This" a subject. Change unneeded capitalization.

File: website/docs/cloud-docs/api-docs/team-members.mdx [hunk 4]
```diff
@@ -44,9 +44,9 @@ The Team Membership API is used to add or remove users from teams. The [Team API
 
 -> **Note:** To add users to a team, they must first be invited to join the organization by email, and accept the invitation. This ensures that you don't accidentally add the wrong person by mistyping a username. See [the Organization Memberships API documentation](/cloud-docs/api-docs/organization-memberships) for more information.
```
Comment (mgarrell777) [inline website/docs/cloud-docs/api-docs/team-members.mdx:45]:
```suggestion
-> **Note:** To add users to a team, they must first receive and accept the invitation to join the organization by email. This process ensures that you do not accidentally add the wrong person by mistyping a username. Refer to [the Organization Memberships API documentation](/cloud-docs/api-docs/organization-memberships) for more information.
```

Suggest rewording the 1st sentence to avoid a passive voice style issue. Suggest adding "process" to avoid a "This" without a subject in the 2nd sentence. Change "See" to "Refer to" to avoid ableist language.

**Review by mgarrell777 — APPROVED** [review]
@Uk1288 Thanks for your efforts on this PR. I have a few small wording change suggestions, but overall it looks good.
